scripts/bump_version.sh

```diff
@@ -18,11 +21,17 @@ if ! GO111MODULE=on go run ./cmd/builder-bumper; then
   exit 1
 fi
 
+## Internal functions
+github_api() {
+  local url
+  url="https://api.github.com/${1}"
+  shift 1
+  curl --retry 5 --silent --fail -u "${GITHUB_USER}:${GITHUB_TOKEN}" "${url}" "$@"
+}
+
 if [[ -n "$(git status --porcelain)" ]]; then
   # Check if a PR is already opened for the branch.
-  prLink=$(curl --show-error --silent \
-    -u "${GITHUB_USER}:${GITHUB_TOKEN}" \
-    "https://api.github.com/repos/prometheus/golang-builder/pulls?head=golang-builder:${BRANCH_NAME}" | jq '.[0].url')
+  prLink=$(github_api "repos/${ORG}/${REPO}/pulls?state=open&head=${ORG}:${BRANCH_NAME}" | jq '.[0].html_url')
```

Comment:
why the change to html_url?

Reply:
`url` returns the `api.github.com` URL. I copied this from [Prometheus sync repo](https://github.com/prometheus/prometheus/blob/022c91c22194655ecdc506d5b1aee55bb7b0e1e0/scripts/sync_repo_files.sh#L208).